### PR TITLE
Focus control and nicer UI for Windows Phone

### PR DESCRIPTION
**Pull request by Gillardo**
Create a Take button as well as having the Cancel button.  Made the buttons "bootstrap" like, but using styles and not css. 

When the cancel button is pressed, the event of "no image selected" is reported, which matches all the other platforms.

Added focus control when the screen is clicked.

### CODE REVIEW

File: src/windows/CameraProxy.js [hunk 1]
```diff
@@ -324,13 +325,20 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
...
+        captureTakePhotoButton.innerText = "Take";
+        captureTakePhotoButton.style.cssText = buttonStyle + "position: fixed; left: 0; bottom: 0; margin: 20px; z-index: 1000";
 
         // Create cancel button
         captureCancelButton = document.createElement("button");
         captureCancelButton.innerText = "Cancel";
-        captureCancelButton.style.cssText = "position: fixed; right: 0; bottom: 0; display: block; margin: 20px; z-index: 1000";
-
+        captureCancelButton.style.cssText = buttonStyle + "position: fixed; right: 0; bottom: 0; margin: 20px; z-index: 1000";
+
```
Comment (muratsu) [inline src/windows/CameraProxy.js]: nit: remove trailing whitespace

File: src/windows/CameraProxy.js [hunk 2]
```diff
@@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
             if (sensor !== null) {
                 sensor.addEventListener("orientationchanged", onOrientationChange);
             }
-            capturePreview.addEventListener('click', captureAction);
+            //capturePreview.addEventListener('click', captureAction);
```
Comment (muratsu) [inline src/windows/CameraProxy.js]: nit: remove comment if not necessary

Comment (Gillardo) [inline src/windows/CameraProxy.js]: Am I suppose to edit my file with these changes? Never had a "nit" before

Comment (muratsu) [inline src/windows/CameraProxy.js]: you can iterate with a new commit, I can merge them. Also before I can merge your changes, you need to sign Apache's Contributor License Agreement (can be done online): http://www.apache.org/licenses/#clas

Comment (Gillardo) [inline src/windows/CameraProxy.js]: I will do another commit for u. Probably tomorrow morning now.

I am already signed up and sent my license off as contributed to other
cordova plug in projects

**Comment by Gillardo** [comment]
Believe i have done what you asked?

**Comment by muratsu** [comment]
Thank you @Gillardo ! Merging your changes.

**Comment by vladimir-kotikov** [comment]
@Gillardo, could you please close this one manually.

**Comment by Gillardo** [comment]
done

**Comment by vladimir-kotikov** [comment]
Thanks!

**Pull request closed without merging**